ts/node/migration/sessionMigrations.ts

```diff
@@ -1582,6 +1584,43 @@ function updateToSessionSchemaVersion30(currentVersion: number, db: BetterSqlite
   console.log(`updateToSessionSchemaVersion${targetVersion}: success!`);
 }
 
+/**
+ * Returns the logged in user priv/public key or null if no user is logged in at the time of migration
+ */
+function getLoggedInUserDuringMigration(db: BetterSqlite3.Database) {
+  const keys = getIdentityKeys(db);
+
+  const userAlreadyCreated = !!keys && !isEmpty(keys.privateEd25519);
+
+  if (!userAlreadyCreated) {
+    return null;
+  }
+
+  return keys;
+}
+
+/**
+ * Returns the logged in user conversation model or null if no user is logged in at the time of migration
+ */
+function getLoggedInUserConvoDuringMigration(db: BetterSqlite3.Database) {
+  const keys = getLoggedInUserDuringMigration(db);
+
+  const userAlreadyCreated = !!keys && !isEmpty(keys.privateEd25519);
+
+  if (!userAlreadyCreated || isEmpty(keys.publicKeyHex)) {
```

Comment:
Can't this just be `if (!keys || isEmpty(keys.publicKeyHex){` and then we can remove line 1608? Since keys would be null in this case